feature1/one-more-file.txt

```diff
@@ -0,0 +1,3 @@
+This file is also added in the feature1 branch under feature folder of root project directory
```

Comment:
this looks perfectly fine and works as expected

Reply:
thanks for your valuable feedback!!